tests/sentry/hybridcloud/test_organization.py

```diff
@@ -60,7 +60,9 @@ def basic_filled_out_org() -> tuple[Organization, list[User]]:
     return org, [owner, other_user]
 
 
-def parameterize_with_orgs(f: Callable):
+def parameterize_with_orgs(
+    f: Callable[[Callable[[], tuple[Organization, list[User]]]], None]
+) -> Callable[[Callable[[], tuple[Organization, list[User]]]], None]:
```

Comment:
```suggestion
WithOrgsCallback = Callable[[Callable[[], tuple[Organization, list[User]]]], None]

def parameterize_with_orgs(f: WithOrgsCallback) -> WithOrgsCallback:
```

The duplicate type could be extracted 🤷